### PR TITLE
get-resources

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: cmd/getResources.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+		prettyJSON, err := json.Pretty(resp.Payload)
+		if err != nil {
+			return err
+		}
+		fmt.Println(*prettyJSON)
+		return nil
+	},
+}
+
+func init() {
```
Comment (nilsgstrabo) [inline cmd/getResources.go:106]: Include flag completion, see restartComponent.go.go for an example

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Include registration of flag completion

File: cmd/getResources.go (hunk 1 above)
Comment (satr) [reply cmd/getResources.go:107]: Added flag

Comment (nilsgstrabo) [reply cmd/getResources.go:107]: Missing flag completion for environment and component

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**